crates/sui-types/src/messages.rs

```diff
@@ -1981,7 +1918,8 @@ impl VerifiedSignedTransaction {
         authority: AuthorityName,
         secret: &dyn Signer<AuthoritySignature>,
     ) -> Self {
-        Self::new_from_verified(SignedTransaction::new(
+        // ok bc transcation is verified? Is this correct?
+        Self::new_unchecked(SignedTransaction::new(
```

Comment:
cc @lxfind this method is used extensively and want to check, not running verification here is safe bc the `transaction: VerifiedTransaction` is already in an verified envelope.

Reply:
VerifiedTransaction is just sender signature verified, but not validator signature verified.
Does SignedTransaction::new do any verifications? If so then it seems new_from_verified is a better name

Reply:
so these two are the same methods https://github.com/MystenLabs/sui/pull/10097/files#r1152373121

Reply:
im ok with changing either name, but want to rename to make sure the caller is ok about bypassing the verification. i guess you are suggesting this should actually run verification?

Reply:
Oh sorry I took it back. SignedTransaction::new signs it so it should be correct by construction.
Ideally this suggests that `Envelope<T, AuthoritySignInfo>::new` constructor shouldn't be in Envelope but should be in VerifiedEnvelope instead, since signing by construction creates a valid envelope

Reply:
So I don't have a strong opinion on the naming here. But ideally in the future we may want to reorganize the API a bit